indico/modules/rb_new/client/styles/palette.scss

```diff
@@ -22,12 +22,10 @@ $highlight-color: #7dcfb6;
 $dark-highlight-color: #45ba97;
 $primary-color: #2185d0;
 
-$sui-red: #db2828;
-$sui-orange: #f2711c;
-$sui-green: #21ba45;
-$sui-grey: #767676;
-
-$light-red: #ffbdbd;
+$sui-red: $red;
```

Comment:
Just out of curiosity, these new `sui` colors are being changed because are the default ones in the new indico theme?

Reply:
Yes. I tried to find a palette that doesn't look too different from the one we use in the rest of Indico and that fits the Semantic UI... ehm... semantics.
The colours are [here](https://github.com/pferreir/indico-sui-theme/blob/master/theme/indico/globals/site.variables#L23).